src/trufont/objects/factories.py

```diff
@@ -76,13 +76,16 @@ def pathGraphicsPathFactory(path):
     points = path._points
     if not points:
         return graphicsPath
+    if all(point.type is None for point in points):
+        # https://github.com/trufont/trufont/pull/599#issuecomment-441391756
+        raise ValueError("All-off-curve paths are not currently supported.")
+
     start = points[0]
     open_ = skip = start.type == "move"
     if open_:
         graphicsPath.MoveToPoint(start.x, start.y)
     else:
         start = points[-1]
-        assert start.type is not None
```

Comment:
Can you bring this back, a path is a suite of segments and a segment ends with an oncurve.

Reply:
Oops. Can't remember why I deleted this.

Reply:
for the "all off curves" thing probably.